src/Layout/Box/Box.test.tsx

```diff
@@ -47,3 +48,15 @@ describe('<Box /> ensure default space theme is overwritten', () => {
     expect(container.firstChild).toHaveStyle('margin: 8px');
   });
 });
+
+describe('<Box /> shadow', () => {
+  it('should match snapshot when shadow down is set', () => {
+    const { asFragment } = render(<Box boxShadow={Shadows[1].down} />);
+    expect(asFragment()).toMatchSnapshot();
+  });
+
+  it('should match snapshot when shadow up is set', () => {
+    const { asFragment } = render(<Box boxShadow={Shadows[4].up} />);
```

Comment:
I'm not sure that I like this interface :thinking: It implies to me that there should be a `Shadow[1].up` too.

Reply:
Ah, good point. 🤔

Reply:
Btw, the `up` and `down` took `ant-design` as reference.

![image](https://user-images.githubusercontent.com/13249748/102776296-76781200-43c9-11eb-9f26-8ca20795847d.png)

https://ant.design/docs/spec/shadow-cn